lms/static/js/edxnotes/plugins/accessibility.js

```diff
@@ -131,19 +131,27 @@ define(['jquery', 'underscore', 'annotator_1.2.9'], function ($, _, Annotator) {
         },
 
         getEditorTabControls: function () {
```

Comment:
This is a very interesting way to manage tab order for a view. Not for this PR, but we should consider sharing this approach more widely.